scripts/asm-installer/asm_vm

```diff
@@ -1246,6 +1246,7 @@ gen_gce_service_proxy() {
     --arg asm_revision "${ASM_META_REVISION}"\
     --arg credential_identity_provider "${CREDENTIAL_IDENTITY_PROVIDER}" \
     --arg network "${NETWORK}" \
+    --arg clusterID "${CLUSTER_NAME}" \
```

Comment:
Does this need to be the same as cluster name in multicluster? https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/blob/956568b25dd2318fec585cfe17539671fa90841b/asm/istio/options/multicluster.yaml#L23

Reply:
Oh, right those are different. Yes it does.

Reply:
So should it be cn-${PROJECT_ID}-${CLUSTER_LOCATION}-${CLUSTER_NAME} here?

Reply:
Fixed.. I don't want to hardcode that templating here since it has variations in other environments (such as onprem). I used something a bit more future proof: grabbing it from the injector values.

Reply:
Just manually tested with these changes